samples/rqd/blender/blender2.79-docker/Dockerfile

```diff
@@ -15,11 +15,11 @@ RUN yum -y install \
 
 # Download and install Blender 2.79
```

Comment:
Let's remove the version number from the comment here -- it's out of date now and it's not necessary anyway.

Reply:
Noted, missed that one. Resolved in https://github.com/AcademySoftwareFoundation/OpenCue/pull/1259/commits/b9191eeb649d7c58369943664f1f144ba3619726

Reply:
Sorry to clarify, I think you can just say "Download and install Blender" -- it's not necessary to include the version number.

Reply:
My bad, I misread that. 😄 
Resolved in: https://github.com/AcademySoftwareFoundation/OpenCue/pull/1259/commits/9422e0382548759042bc564247142034aa415da5